sklearn/ensemble/tests/test_voting_classifier.py

```diff
@@ -0,0 +1,49 @@
+"""Testing for the boost module (sklearn.ensemble.boost)."""
+
+import numpy as np
+
+from sklearn.utils.testing import assert_almost_equal
+from sklearn.utils.testing import assert_raises, assert_raises_regexp
+
+from sklearn.linear_model import LogisticRegression
+from sklearn.naive_bayes import GaussianNB 
+from sklearn.ensemble import RandomForestClassifier
+from sklearn.ensemble import VotingClassifier
+from sklearn import datasets
+from sklearn import cross_validation
+
+
+# Load the iris dataset and randomly permute it
+iris = datasets.load_iris()
+X, y = iris.data[:, 1:3], iris.target
+
+
+def test_majority_label_iris():
+    """Check classification by majority label on dataset iris."""
+    
+    np.random.seed(123)    
+    clf1 = LogisticRegression()
+    clf2 = RandomForestClassifier()
+    clf3 = GaussianNB()
+    eclf = VotingClassifier(estimators=[clf1, clf2, clf3], voting='hard')
+    scores = cross_validation.cross_val_score(eclf, X, y, cv=5, scoring='accuracy')
+    assert_almost_equal(scores.mean(), 0.94, decimal=2)
+    
+
+def test_weights_iris():
+    """Check classification by average probabilities on dataset iris."""
+    
+    np.random.seed(123)    
+    clf1 = LogisticRegression()
+    clf2 = RandomForestClassifier()
+    clf3 = GaussianNB()
+    eclf = VotingClassifier(estimators=[clf1, clf2, clf3], voting='soft', weights=[1,2,10])
+    scores = cross_validation.cross_val_score(eclf, X, y, cv=5, scoring='accuracy')
+    assert_almost_equal(scores.mean(), 0.93, decimal=2)   
+
+
```

Comment:
In general, could you write tests checking for the correctness of the results? (E.g., by comparing the vectors returned by `predict` or `predict_proba` with the ones that you would manually obtain on a toy problem?)